app/views/collection_drafts/forms/_model_errors.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if get_resource.errors.any? %>
+  <!-- Model Errors Flag -->
+  <section class="errors">
```

Comment:
is this class used for anything besides the check commented on above? If so, can we add a more specific class to use for that check, i.e. `model-errors`?

Reply:
Done